packages/browser/src/backend.ts

```diff
@@ -61,6 +61,7 @@ export class BrowserBackend extends BaseBackend<BrowserOptions> {
     const transportOptions = {
       ...this._options.transportOptions,
       dsn: this._options.dsn,
+      envelopeTunnel: this._options.envelopeTunnel,
```

Comment:
I would prefer if we call this option only `tunnel`.
Users don't care and understand what an envelope is and I would consider it an implementation detail.

Reply:
I’m fine calling it just tunnel.

Reply:
Updated